engines/agos/metaengine.cpp

```diff
@@ -218,19 +218,33 @@ Common::KeymapArray AgosMetaEngine::initKeymaps(const char *target) const {
 	using namespace AGOS;
 
 	Common::String gameId = ConfMan.get("gameid", target);
-
+	
 	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "agos-main", _("AGOS main"));
-	Keymap *gameKeyMap = new Keymap(Keymap::kKeymapTypeGame, gameId, _("Game Keymappings"));
+	Keymap *gameKeyMap = new Keymap(Keymap::kKeymapTypeGame, "game-shortcuts", _("Game Keymappings"));
+	Keymap *gameKeyMap2 = new Keymap(Keymap::kKeymapTypeGame, "game-shortcuts-2", _("Game Keymappings 2"));
 	Action *act;
 
-	act = new Action("Pause", _("Pause"));
+	act = new Action(kStandardActionLeftClick, _("Left Click"));
+	act->setLeftClickEvent();
+	act->addDefaultInputMapping("MOUSE_LEFT");
+	act->addDefaultInputMapping("JOY_LEFT_SHOULDER");
```

Comment:
I suggest sticking to the default game keymap from MetaEngine::initKeymaps of JOY_A for left click and JOY_B for right click - same goes for most of the defaults from that list if copied from that default game keymap.